configure.in

```diff
@@ -7982,7 +7990,6 @@ EXTRALIBS="$LDFLAGS $LDFLAGS_VERSIONING $LIBS $DMALLOC_LIBS"
 EXTRALIBS_XML="$EXPAT_LINK"
 EXTRALIBS_HTML="$MSPACK_LINK"
 EXTRALIBS_MEDIA="$GST_LIBS"
-EXTRALIBS_STC="-lwxscintilla${WX_LIB_FLAVOUR}-${WX_RELEASE}${HOST_SUFFIX}"
```

Comment:
Sorry, I don't understand: why don't we need this any longer?

Reply:
This variable is only set but never used before. You can search for `EXTRALIBS_STC` in Makefile.in and there is no result. It is used in order to link against imm32 after the change I made in:
https://github.com/wxWidgets/wxWidgets/pull/1852/commits/f730da3c14927b1833fc776479bd457b494bb30e#diff-0c9f5c5268aea486bb1cf4ce87975c38R414

STC does require wxScintilla, but it is already done in https://github.com/wxWidgets/wxWidgets/blob/master/build/bakefiles/multilib.bkl#L395